ts/nni_manager/training_service/reusable/environments/dlcEnvironmentService.ts

```diff
@@ -53,8 +54,11 @@ export class DlcEnvironmentService extends EnvironmentService {
     public get getName(): string {
         return 'dlc';
     }
-
+    public sleep(time: any): Promise<any> {
+        return new Promise((resolve) => setTimeout(resolve, time*1000));
```

Comment:
Seems you are re-implementing `timersPromises.setTimeout()`.
https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options

Reply:
yes, I will just use it